packages/@sanity/migrate/src/mutations/transaction.ts

```diff
@@ -5,7 +5,17 @@ export interface Transaction {
   id?: string
   mutations: Mutation[]
 }
-export function transaction(id: string, mutations: Mutation[]): Transaction
+
+/**
+ * Wraps a set of mutations in a transaction.
+ * Note: use with caution. Transactions cannot be optimized and will be submitted one-by-one, which means they will make
+ * your migration run slower and produce more API requests.
+ * @param transactionId - The transaction id. This is optional and should usually be omitted, as it will be auto-generated by the server if not provided.
```

Comment:
```suggestion
 * @param transactionId - The transaction ID. This is optional and should usually be omitted, as it will be auto-generated by the server if not provided.
```